numpy/ma/tests/test_core.py

```diff
@@ -1723,6 +1723,17 @@ def setUp(self):
         xm[2] = masked
         self.intdata = (x, y, xm)
         self.floatdata = (x.astype(float), y.astype(float), xm.astype(float))
+        self.othertypes = (
```

Comment:
I was going to ask if there was a better way to handle this. Is there one for complex, as well?